query_optimizer/tests/execution_generator/CMakeLists.txt

```diff
@@ -17,39 +17,39 @@ add_test(quickstep_queryoptimizer_tests_executiongenerator_create
          "../quickstep_queryoptimizer_tests_ExecutionGeneratorTest"
          "${CMAKE_CURRENT_SOURCE_DIR}/Create.test"
          "${CMAKE_CURRENT_BINARY_DIR}/Create.test"
-         "${CMAKE_CURRENT_BINARY_DIR}/")
+         "${CMAKE_CURRENT_BINARY_DIR}/Create/")
```

Comment:
@jianqiao What's bug then? Each `quickstep_queryoptimizer_tests_ExecutionGeneratorTest` should clean up the environment after running.